### PR TITLE
PCM-1693

### PR DESCRIPTION
**Pull request by maxwellmooney13**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,18 @@
...
+      "patterns": ["src"],
+      "extensions": "ts",
+      "ignore": "node_modules/**",
+      "legacyWatch": true,
+      "delay": 2000
+    }
+  },
   "scripts": {
+    "watchy": "npm run build:src && npm run build:module",
+    "watch": "npm-watch watchy",
```
Comment (djhouseknecht) [inline package.json:32]: A nice little script to have hot reloading for the sdk while it is linked 🔥

File: src/media/media.ts [hunk 2]
```diff
@@ -264,8 +264,8 @@ export class SdkMedia extends (EventEmitter as { new(): StrictEventEmitter<Event
    * Note: if using `retryOnFailure` it is recommended to check the media
    *  returned to ensure you received the desired device.
    *
-   * Warning: if `sdk.media.requestPermissions(type)` has NOT been called before
-   *  calling `startMedia`, `startMedia` will call `sdk.media.requestPermissions(type)`.
+   * Warning: if `sdk.media.requestMediaPermissions(type)` has NOT been called before
+   *  calling `startMedia`, `startMedia` will call `sdk.media.requestMediaPermissions(type)`.
```
Comment (djhouseknecht) [inline src/media/media.ts:268]: Thank you for correcting these

**Review by djhouseknecht — COMMENTED** [review]
See requested changes comment

**Review by djhouseknecht — CHANGES_REQUESTED** [review]
Two things: 
1. You have an `.npmrc` somewhere in your install tree because the `package-lock.json` installed using our private registry and not `registry.npmjs`. 
2. merge conflicts in `media.ts`

**Comment by zservies** [comment]
I'm not sure how urgent these changes are but this will have conflicts with the PR I have up for enabling softphone calls so it'll probably have to wait until that gets merged.

File: changelog.md [hunk 3]
```diff
@@ -4,6 +4,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v6.0.1...HEAD)
+
+### Fixed
+* [PCM-1693](https://inindca.atlassian.net/browse/PCM-1693) - Fixed `client.ts` file by adding in check for `audioTracks` size as well as passing in track in question that one is trying to remove.
```
Comment (jensengar) [inline changelog.md:9]: Can we reword this please? It's a bit hard to read and this is customer facing.

**Review by jensengar — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 3 above)
Comment (maxwellmooney13) [reply changelog.md]: Absolutely. I'm sorry, I didn't realize this was customer facing.

**Review by djhouseknecht — CHANGES_REQUESTED** [review]
I'm still seeing our `purecloud.jfrog.io` registry in the package-lock.json. You may need to delete the file and do a fresh install. Make sure you don't have a `.npmrc` file in your sdk root folder structure.

**Review by djhouseknecht — APPROVED** [review]

File: changelog.md [hunk 4]
```diff
@@ -4,6 +4,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v6.0.1...HEAD)
+
+### Fixed
```
Comment (zservies) [inline changelog.md:8]: Move this section below PCM-1669. PCM-1669 isn't a bug fix its a new feature and belongs under "Added".

**Review by zservies — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 4 above)
Comment (maxwellmooney13) [reply changelog.md:8]: Oh sorry about that! That was a merge conflict resolution and I didn't realize that happened

**Review by zservies — APPROVED** [review]

**Review by jensengar — APPROVED** [review]

**Pull request merged**